docs/account_management/gitlab_user.md

```diff
@@ -1,31 +1,57 @@
-# UAB GitLab Registration Instructions
+# UAB GitLab Overview
+
+Welcome to UAB GitLab! This is a UAB-specific [GitLab](https://about.gitlab.com/). GitLab is similar to [GitHub](https://github.com/), but hosted here at UAB on secure servers. 
+
+### Researcher GitLab Use Cases
+GitLab can be used:
+
+   - For reproducibility 
+     - Analysis and software code can be kept in one, central repository everyone can use instead of spread across multiple computers/places.
+     - Code can be versioned and tracked as it changes over time.
+     - Software versions can be recorded, virtual environments can be documented, and containers can be recorded to help future-proof analyses.
+   - Collaboration
+     - GitLab is a central place to create code, edit, and track needed code changes (issues) with your lab and collaborators.
+     - Multiple people can use, modify, and merge changes in code while communicating with the broader team all along the way.
+   - Security
+     - Unlimited private repositories for internal code projects.
+     - Set behind UAB authentication.
+
+### Software Developers (and Researchers!) GitLab Use Cases
+UAB GitLab is useful for software developers. It is a single application for the entire software development lifecycle. From project planning and source code management to continuous integration (CI) and continuous deployment (CD), monitoring, and security.
```

Comment:
I think it's best practice to have a line break between the subheading and the text body

Reply:
Got it. All the white space messes with me, but I'll do that for the future.